<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="imageDisplayer">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConvert">
    <property name="title">
     <string>&amp;Convert to</string>
    </property>
    <addaction name="actionGrayscale"/>
    <addaction name="separator"/>
    <addaction name="actionBinary"/>
    <addaction name="actionConvolved"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionHistogram"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuAlgebraicOp">
     <property name="title">
      <string>&amp;Algebraic Operation</string>
     </property>
     <addaction name="actionAddition"/>
     <addaction name="actionSubtraction"/>
     <addaction name="actionMultiplication"/>
     <addaction name="actionDivision"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionPointOp"/>
    <addaction name="menuAlgebraicOp"/>
    <addaction name="actionGeometricOp"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>F&amp;ilters</string>
    </property>
    <addaction name="actionGaussian"/>
    <addaction name="actionMean"/>
    <addaction name="actionMedian"/>
   </widget>
   <widget class="QMenu" name="menuMorphology">
    <property name="title">
     <string>&amp;Morphology</string>
    </property>
    <widget class="QMenu" name="menu_Reconstruct">
     <property name="title">
      <string>Re&amp;construct</string>
     </property>
     <addaction name="actionReconstructOBR"/>
     <addaction name="actionReconstructCBR"/>
    </widget>
    <addaction name="actionMorphologyOp"/>
    <addaction name="separator"/>
    <addaction name="actionDistanceTransform"/>
    <addaction name="actionSkeleton"/>
    <addaction name="actionRestore"/>
    <addaction name="separator"/>
    <addaction name="actionEdgeDetection"/>
    <addaction name="actionGradient"/>
    <addaction name="menu_Reconstruct"/>
   </widget>
   <widget class="QMenu" name="menu_Texture">
    <property name="title">
     <string>&amp;Texture</string>
    </property>
    <addaction name="actionSynthesize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuConvert"/>
   <addaction name="menuFilters"/>
   <addaction name="menuMorphology"/>
   <addaction name="menu_Texture"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>&amp;Grayscale Image</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>&amp;Histogram</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="text">
    <string>&amp;Binary Image</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionConvolved">
   <property name="text">
    <string>&amp;Convolved Image</string>
   </property>
  </action>
  <action name="actionPointOp">
   <property name="text">
    <string>&amp;Point Operation</string>
   </property>
  </action>
  <action name="actionGeometricOp">
   <property name="text">
    <string>&amp;Geometric Operation</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>&amp;Gaussian Filter</string>
   </property>
  </action>
  <action name="actionMean">
   <property name="text">
    <string>&amp;Mean Filter</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>M&amp;edian Filter</string>
   </property>
  </action>
  <action name="actionAddition">
   <property name="text">
    <string>&amp;Addition</string>
   </property>
  </action>
  <action name="actionSubtraction">
   <property name="text">
    <string>&amp;Subtraction</string>
   </property>
  </action>
  <action name="actionMultiplication">
   <property name="text">
    <string>&amp;Multiplication</string>
   </property>
  </action>
  <action name="actionDivision">
   <property name="text">
    <string>&amp;Division</string>
   </property>
  </action>
  <action name="actionMorphologyOp">
   <property name="text">
    <string>&amp;Morphology Operations</string>
   </property>
  </action>
  <action name="actionDistanceTransform">
   <property name="text">
    <string>&amp;Distance Transform</string>
   </property>
  </action>
  <action name="actionSkeleton">
   <property name="text">
    <string>Extract &amp;Skeleton</string>
   </property>
  </action>
  <action name="actionEdgeDetection">
   <property name="text">
    <string>&amp;Edge Detection</string>
   </property>
   <property name="toolTip">
    <string>Edge Detection</string>
   </property>
  </action>
  <action name="actionGradient">
   <property name="text">
    <string>&amp;Gradient</string>
   </property>
  </action>
  <action name="actionReconstructOBR">
   <property name="text">
    <string>&amp;OBR</string>
   </property>
  </action>
  <action name="actionReconstructCBR">
   <property name="text">
    <string>&amp;CBR</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>&amp;Restore from Skeleton</string>
   </property>
  </action>
  <action name="actionSynthesize">
   <property name="text">
    <string>&amp;Synthesize</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
